js/views/modals/Settings/Page.js

```diff
@@ -28,39 +28,50 @@ export default class extends baseVw {
     this.listenTo(this.profile, 'sync', () => app.profile.set(this.profile.toJSON()));
   }
 
-  avatarLeftClick() {
+  avatarRotate(direction) {
     if (this.avatarCropper.cropit('imageSrc')) {
-      this.avatarCropper.cropit('rotateCCW');
+      // we don't have to check the rotated size of the avatar, because it is square
+      this.avatarCropper.cropit(direction > 0 ? 'rotateCW' : 'rotateCCW');
       this.avatarChanged = true;
     }
   }
 
+  avatarLeftClick() {
+    this.avatarRotate(-1);
+  }
+
   avatarRightClick() {
-    if (this.avatarCropper.cropit('imageSrc')) {
-      this.avatarCropper.cropit('rotateCW');
-      this.avatarChanged = true;
-    }
+    this.avatarRotate(1);
   }
 
-  headerLeftClick() {
+  headerRotate(direction) {
     if (this.headerCropper.cropit('imageSrc')) {
-      this.headerCropper.cropit('rotateCCW');
+      const loadedSize = this.headerCropper.cropit('imageSize');
```

Comment:
The block to determine whether an avatar/header is too small is repeated. Could we break that determination off into a separate function? It's possible we'll need to determine this elsewhere in the view and I wouldn't want to repeat the block again.

e.g:

```
  headerTooSmall() {
    const loadedSize = this.headerCropper.cropit('imageSize');

    return (loadedSize.height < this.headerMinWidth ||
        loadedSize.width < this.headerMinHeight);
  }
```